dnf.spec

```diff
@@ -333,6 +333,12 @@ popd
 %endif
 
 %changelog
+* Mon Feb 13 2017 Jaroslav Mracek <jmracek@redhat.com>
```

Comment:
Changelog should be modified with release. Also there are not-related-to-this-pr items. Remove this please.